ios-starter/AppDelegate.swift

```diff
@@ -15,7 +15,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 	
 	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
 		// Override point for customization after application launch.
-		return true
-	}
+		return true}
 }
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/omirho/ios-starter/blob/85b34c420d3ac37951f69b4474dc2286502bc47b/ios-starter/AppDelegate.swift#L20"></span>Files should have a single trailing newline.</td>
    </tr>
  </tbody>
</table>